app/views/user_mailer/unlock_instructions.html.erb

```diff
@@ -1,5 +1,3 @@
-<p>Hello <%= @resource.name %>,</p>
```

Comment:
The layout refers to `@user` now. Is this going to cause a problem? I'm not sure about test coverage here, did you check?